client/my-sites/plans/current-plan/index.jsx

```diff
@@ -329,5 +334,6 @@ export default connect( ( state, { requestThankYou } ) => {
 		showThankYou: requestThankYou && isJetpackNotAtomic,
 		scheduleId: getConciergeScheduleId( state ),
 		eligibleForProPlan,
+		isJetpack,
```

Comment:
This will also flag atomic sites as Jetpack sites and the Modernized header will not be visible. 

![image](https://user-images.githubusercontent.com/3422709/217307294-5c8978fd-d533-401e-a696-13c1f9793099.png)


I think the flag on line 318 accounts for this and can be used for our use case as well.

https://github.com/Automattic/wp-calypso/blob/fd4d73e86ebc9dc8af48c2fd0134406ae7b4094c/client/my-sites/plans/current-plan/index.jsx#L318

Reply:
Ah! Good catch. totally ignored this.

Reply:
(and funnily enough, I was testing with an atomic site to confirm the jetpack condition at some point. so I'm the worst! 😆 )